skietbaan_fe/src/components/ScoreCapture.js

```diff
@@ -312,25 +335,26 @@ export default class search extends Component {
     let competitionItem = [];
     if (this.state.competitionsList && this.state.competitionsList.length > 0) {
       for (let i = 0; i < this.state.competitionsList.length; i++) {
-        competitionItem.push(<div className="competition-item-container"> <div key={'mykey' + i}
+        competitionItem.push(<div className="competition-item-container" key={'mykey' + i}> <div key={'mykey' + i}
           className={this.state.clicked != null && this.state.clicked === i ? "active competition-item" : "competition-item"}>
           <li onClick={() => this.CompetitionClicked(i, this.state.competitionsList[i].name)}>
             {this.state.competitionsList[i].name} </li></div></div>);
 
       }
     }
-
+    if(!getCookie("token")){
+      window.location = "/register-page";
```

Comment:
Other paths were camelCase please be consistent

Reply:
changed path name